src/test/java/com/wikia/webdriver/common/remote/operations/CreateCategory.java

```diff
@@ -13,27 +15,31 @@
 public class CreateCategory {
 
   private static final int PARENT_ID = 1;
-
   public static final String CREATE_CATEGORY_URL_SUFFIX = "%s/forums";
-
   private final PostRemoteOperation remoteOperation;
 
   CreateCategory(User user) {
     remoteOperation = new PostRemoteOperation(user);
   }
 
-  public CategoryPill.Data execute(final CreateCategoryContext context) {
+  public CategoryPill.Data execute(final CreateCategoryContext context) throws RuntimeException {
     JSONObject jsonObject = new JSONObject(ImmutableMap.builder()
         .put("name", context.getCategoryName())
         .put("parentId", PARENT_ID)
         .put("siteId", context.getSiteId())
         .build());
 
-    final String response = remoteOperation.execute(buildUrl(context), jsonObject);
+    String response;
+    try {
+      response = remoteOperation.execute(buildUrl(context), jsonObject);
+    } catch(RemoteException e) {
+      PageObjectLogging.logError("error: ", e);
+      throw new RuntimeException("Could not create a new category.", e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00112)